### PR TITLE
Apply continuous queries on all metric type except timing

### PR DESCRIPTION
**Pull request by amitbose327**
@jordmoz and @ankanm Please review.

### CODE REVIEW

File: ansible/roles/monitoring-influxdb9/templates/influxdb.retention.j2 [hunk 1]
```diff
@@ -10,7 +10,7 @@
   influx -execute "drop continuous query downsample_1m  on {{ database }}"
   influx -execute "drop continuous query downsample_10m on {{ database }}"
 
-  influx -execute "CREATE CONTINUOUS QUERY downsample_1m  on {{ database }} begin SELECT last(value) as last, mean(value) as value into  _1m.:MEASUREMENT from  /.*/ group by time(1m), * END"
-  influx -execute "CREATE CONTINUOUS QUERY downsample_10m on {{ database }} begin SELECT last(last)  as last, mean(value)  as value into _10m.:MEASUREMENT from  _1m./.*/ group by time(10m), * END"
+  influx -execute "CREATE CONTINUOUS QUERY downsample_1m  on {{ database }} begin SELECT last(value) as last, mean(value) as value into  _1m.:MEASUREMENT from  /.*/ where metric_type!='timing' group by time(1m), * END"
```
Comment (jordmoz) [inline ansible/roles/monitoring-influxdb9/templates/influxdb.retention.j2]: space character between metric_type and != ?

File: ansible/roles/monitoring-influxdb9/templates/influxdb.retention.j2 [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
+  influx -execute "CREATE CONTINUOUS QUERY downsample_10m on {{ database }} begin SELECT last(last)  as last, mean(value)  as value into _10m.:MEASUREMENT from  _1m./.*/ where metric_type!='timing' group by time(10m), * END"
```
Comment (jordmoz) [inline ansible/roles/monitoring-influxdb9/templates/influxdb.retention.j2]: space character between metric_type and != ?

**Comment by jordmoz** [comment]
2 minor comments, but LGTM apart from that.

**Comment by amitbose327** [comment]
Added spaces according to Jord's comments.

**Comment by ankanm** [comment]
LG

**Pull request merged**